.github/workflows/static.yml

```
@@ -35,11 +35,13 @@ jobs:
           python-version: '3.8'
           architecture: 'x64'
       - run: python -m pip install --upgrade pip setuptools jsonschema
-      - run: pip install -e .[pylint,pycodestyle,pyflakes]
+      # If we don't install pycodestyle, pylint with throw an unused-argument error in pylsp/plugins/pycodestyle_lint.py:72
+      # This error cannot be resove by adding a pylint: disable=unused-argument comment ...
+      - run: pip install -e .[pylint,black,pycodestyle,pyflakes]
```

```suggestion
      - run: |
          pip install -e .[pylint,pycodestyle,pyflakes]
          pip install black
```

We need to install Black directly here.